legion/engine/physics/halfedgeface.hpp

```diff
@@ -20,12 +22,22 @@ namespace legion::physics
             concave
         };
 
-        HalfEdgeEdge* startEdge;
-        math::vec3 normal;
-        math::vec3 centroid;
+		math::vec3 normal;
+		math::vec3 centroid;
```

Comment:
strange indentation?

Reply:
![image](https://user-images.githubusercontent.com/37859544/151352986-e3cee47b-c1fd-464e-b429-8a988a32e1c3.png)
I'm not sure why git detected a change

Reply:
perhaps there's a spaces/tabs issue

Reply:
yeah make sure to run `tools/cleanup.bat`